### ISSUE TITLE
Properly return focus after managing badge/global-badge's skills

### ISSUE THREAD

**Issue by sudo-may** [NationalSecurityAgency/skills-service#2843]
- After adding or removing skill to/from a badge focus shall be placed on the Select Skills dropdown
- If delete operation is cancelled focus should be placed on the Delete button

**Comment by sudo-may** [comment]
pr: https://github.com/NationalSecurityAgency/skills-service/pull/2866

**Comment by rmmayo** [comment]
:+1:
